app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const app = express()
+
+//---Controller Functions---
```

Comment:
Comments should be removed before you submit/merge a pull request. There should be no comments in production code (i.e. the code that lives on the main branch!)